packages/ui-components/src/components/labels-dropdown-input-item/labels-dropdown-input-item.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, Event, EventEmitter, Prop, h } from '@stencil/core';
+import { LabelsInputItem, LabelsInputItemEvents } from './labels-dropdown-input-item.types';
+import { EIconName } from '../icon/icon.types';
+
+@Component({
+	tag: 'kv-labels-dropdown-input-item',
+	styleUrl: 'labels-dropdown-input-item.scss',
+	shadow: true
+})
+export class KvLabelsDropdownInputItem implements LabelsInputItem, LabelsInputItemEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) value!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) label!: string;
+
+	/** @inheritdoc */
+	@Event() removeClick: EventEmitter<string>;
```

Comment:
This should be `clickItem`. The remove logic is related to the parent. Furthermore, to make this component reusable we should enable the icon to be configurable and export the icon part.